pom.xml

```diff
@@ -772,5 +772,17 @@
             <artifactId>encoder</artifactId>
             <version>1.2.3</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
It looks the 2 new dependencies are not used. They should be removed IMO.

Reply:
Both dependencies used in SecurityConfiguration.java 
by "import org.springframework.boot.jdbc.DataSourceBuilder;" (line 55)

Reply:
steve is not using spring boot though. it is spring app, not a spring boot app.

Reply:
@fnkbsi steve uses jooq for the data access. You should be able to replace spring-jdbc by that. Then no need to datasourcebuilder.

Reply:
@goekay any feedback about the feature itself?